src/extension/src/azure/azureServices.ts

```diff
@@ -77,7 +77,7 @@ export class AzureServices extends WizardServant {
 
   private static usersFunctionSubscriptionItemCache: SubscriptionItem;
   private static usersCosmosDBSubscriptionItemCache: SubscriptionItem;
-  private static userAppServiceSubsctiptionItemCache: SubscriptionItem;
+  private static usersAppServiceSubsctiptionItemCache: SubscriptionItem;
```

Comment:
changed this variable name so that it is consistent with Functions and Cosmos DB

Reply:
Should we change the name from usersAppService**Subsctiption** to usersAppService**Subscription**?